todo_app/templates/index.html

```diff
@@ -13,14 +13,16 @@ <h5 class="mb-1">{{ item.name }}</h5>
     {% endif %}
     <div><span class="badge {{ badge_style }}">{{ item.status }}</span></div>
   </div>
-  {% if item.status == 'Done' %}
-    <a href="{{ url_for('uncomplete_item', id=item.id) }}" class="btn btn-sm btn-secondary">Mark as Incomplete</a>
-  {% elif item.status == 'To Do' %}
-    <a href="{{ url_for('start_item', id=item.id) }}" class="btn btn-sm btn-success">Start</a>
-  {% else %}
-    <a href="{{ url_for('complete_item', id=item.id) }}" class="btn btn-sm btn-primary">Complete</a>
+  {% if not read_only %}
```

Comment:
Very minor point, but you've got an necessary double negative here - `read_only` is defined as `not user.is_writer()`. And with more complicated levels of auth, complication like this could lead to mistakes.

You could have just set an `is_writer` property in the first place.